game-app/game-core/src/main/java/org/triplea/game/startup/SetupModel.java

```diff
@@ -45,4 +48,17 @@ static void clearPbfPbemInformation(final GameProperties properties) {
     properties.set(IEmailSender.RECIPIENTS, null);
     properties.set(IEmailSender.POST_AFTER_COMBAT, null);
   }
+
+  /** Helper method to be used by implementations to avoid duplicated code. */
+  static boolean canGameStartHelper(GameSelectorModel gameSelectorModel, ServerModel model) {
```

Comment:
The name is confusing, because it can be read as "can the game start the helper"? Suggest removing "Helper" from the name, since you already have it in the comment.

Reply:
I have to change the name a bit to avoid the naming collision with the non static method, but I changed it to `staticCanGameStart` so it is a little bit clearer.